modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding on user-provisioned infrastructure in AWS
+
+You can use forwarding with OpenShift DNS to resolve name queries for your other internal devices using the DNS servers in your data center.
+
+[NOTE]
+====
+The DNS Operator prepends the configuration of `Servers` to `configmap/dns-default`, instead of the entire ConfigMap being statically defined.
+====
+
+.Prerequisites
+
+* AWS user-provisioned infrastructure
+
+.Procedure
+
+. Remove the default ConfigMap asset and code from the `manifests.pkg` file.
++
+. Modify the DNS ConfigMap to create a ConfigMap with additional server configuration blocks based on `Server`. Each `Server` has a forwarding configuration and uses `/etc/resolv.conf` as a resolver of last resort.
```

Comment:
The user modifies the `dns.operator` object named `default`; the operator creates and updates the `ConfigMap` named `dns-default`.  The phrase "last resort" seems a little strong; if none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.